build.proj

```diff
@@ -42,9 +42,9 @@
 
   <Target Name="ValidateAllProjectDependencies"
           Condition="'$(ValidatePackageVersions)'=='true' and '@(ProjectJsonFiles)'!=''">
-    <ValidateProjectDependencyVersions ProjectJsons="@(ProjectJsonFiles)"
+<!--    <ValidateProjectDependencyVersions ProjectJsons="@(ProjectJsonFiles)"
```

Comment:
I find it generally useful to put a comment over commented-out sections explaining why.